cellprofiler_core/bioformats/formatreader.py

```diff
@@ -0,0 +1,19 @@
+#TODO: unimplemented until CellProfiler/CellProfiler#4684 is resolved
```

Comment:
Can we comment the file out then for now?

Reply:
I tried to only comment out omero-related code that could be reached somehow. If there's anything omero related that's not commented out, it is (if I did it correctly) unreachable via any code path. The reason I didn't comment out everything is just because it's surprisingly (to me) a lot. I actually wanted to delete it all, and reconstruct it where needed, but thought better of it.

I could comment it all out if you think it's important or helpful to do so.

Reply:
My only thinking around commenting it out is 
a) in case a convoluted code path finds it, I'd rather know sooner than later (not that CellProfiler contains convoluted code paths, OF COURSE) 
and
b) on the off chance someone (aka future-me) is trying to read and debug this without realizing it's all destined for the trash bin anyway, we'll save them that time.

I don't feel super strongly about it though if you'd really rather not.